packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/usePackSizeController.test.tsx

```diff
@@ -6,30 +6,38 @@ import {
   InvoiceLineNodeType,
   renderHookWithProvider,
 } from '@openmsupply-client/common';
+import { createPlaceholderRow } from './useDraftOutboundLines';
```

Comment:
slight tidy up and failing tests (which are now not failing!)

Reply:
the number of tests shows how complex this thing is 😮  no wonder there were some edge case bugs!